hudi-common/src/main/java/org/apache/hudi/common/config/HoodieReaderConfig.java

```diff
@@ -51,4 +51,11 @@ public class HoodieReaderConfig {
       .sinceVersion("0.13.0")
       .withDocumentation("New optimized scan for log blocks that handles all multi-writer use-cases while appending to log files. "
           + "It also differentiates original blocks written by ingestion writers and compacted blocks written log compaction.");
+
+  public static final ConfigProperty<Boolean> FILE_GROUP_READER_ENABLED = ConfigProperty
+      .key("hoodie.file.group.reader.enabled")
+      .defaultValue(true)
+      .markAdvanced()
+      .sinceVersion("1.0.0")
```

Comment:
Does it deserve to be a config option?

Reply:
Does this option exist because we do not have enough confidence that the new reader would cover all the read use cases? Or not sure it is robust enough? I see the default value is true, that means user would anyway encounter these problems, can we address them and just remove this option?

Reply:
I put it as true just for testing purpose, i will make it false. Sorry for the confusion.